src/tools/miri/tests/pass/float.rs

```diff
@@ -139,20 +183,54 @@ where
 
 fn basic() {
     // basic arithmetic
+    #[cfg(not(bootstrap))]
```

Comment:
Tests never have `bootstrap` set so this is futile.

But does this mean stage 0 tests will be broken when this PR lands? That would not be great for Miri development.

Reply:
> Tests never have `bootstrap` set so this is futile.

I think I realized that after realizing I did it wrong and it still compiled :)

> But does this mean stage 0 tests will be broken when this PR lands? That would not be great for Miri development.

I think stage 0 anything that requires the `f16` or `f128` gates will not work, hence the bootstrap flag everywhere. But I don't know enough about how the tests actually work to know whether or not this will be a problem here, so I am open to suggestions.

Reply:
The important part is that `./x.py test miri --stage 0` still works.

If that does nit work, we might have to add bootstrap support to Miri's compiletest...